.env.example

```diff
@@ -161,3 +162,6 @@ BIBLEIS_DEPRECATE_FROM_VERSION=
 GIDEONS_DEPRECATE_FROM_VERSION=
 
 DOWNLOAD_ACCESS_GROUP_LIST=
+
+BIBLEIS_COMPATIBILITY_APP_NAME='Bible.is'
+GIDEONS_COMPATIBILITY_APP_NAME='Gideon Bible App'
```

Comment:
These 4 new env parameters should be created into the prod environment.

Reply:
thank you. these are done